src/components/game.js

```diff
@@ -0,0 +1,66 @@
+import React from "react"
+// import Button from "./button"
+
+class Game extends React.Component{
+    state={
+        currentScore:0,
+        currentStep:1,
+        gameWon: "",
+        showWin:"none"
+    }
+    handleStep=()=>{
```

Comment:
It would be better if you changed the state through the callback that takes in prevState. 

ex:`this.setState((prevState, props)=> currentScore: prevState.currentScore + prevState.currentStep)`

Doing it the way i just mentioned insures that we don't mess with your current state before setting the new state.